components/Header.vue

```diff
@@ -1,8 +1,13 @@
 <template>
 	<header>
 		<Brand />
-		<NavBar />
-		<SearchForm />
+		<div class="site-nav">
+			<button id="menuToggleButton" @click="toggleNavMenu()" href="javascript:void(0);" aria-expanded="false">
+				&#9776;&nbsp;Menu
```

Comment:
VoiceOver reads this as 'Button, trigram for heaven menu'! I think we should create an SVG for the menu icon instead of using this character.